### PR TITLE
Use d.g.o and remove mold

### PR DESCRIPTION
**Pull request by A6GibKm**
No strong opinion about mold, we just don't use in other apps in Flathub as far as I am aware.

### CODE REVIEW

**Comment by A6GibKm** [comment]
Will remove wip status once I test it.

File: org.freedesktop.Bustle.json [hunk 1]
```diff
@@ -16,13 +15,7 @@
...
     ],
     "build-options": {
-        "append-path": "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm16/bin",
-        "env": {
-            "CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER": "clang",
-            "CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUSTFLAGS": "-C link-arg=-fuse-ld=/usr/lib/sdk/rust-stable/bin/mold",
-            "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER": "clang",
-            "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_RUSTFLAGS": "-C link-arg=-fuse-ld=/usr/lib/sdk/rust-stable/bin/mold"
-        }
+        "append-path": "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm16/bin"
```
Comment (SeaDve) [inline org.freedesktop.Bustle.json:18]: `/usr/lib/sdk/llvm16/bin` I think should also be safe to remove now

```suggestion
        "append-path": "/usr/lib/sdk/rust-stable/bin"
```

Comment (A6GibKm) [reply org.freedesktop.Bustle.json]: Good catch, thanks!

**Comment by A6GibKm** [comment]
Build fails, because its missing metainfo commits. Lets just wait until the release in a week.

**Comment by A6GibKm** [comment]
Closed in favor of #17.

**Pull request closed without merging**